bin/runtime-common/src/lib.rs

```diff
@@ -19,7 +19,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 pub mod messages;
-pub mod messages_api;
 pub mod messages_benchmarking;
 
 #[cfg(feature = "integrity-test")]
```

Comment:
Do we still need `integrity` mod?

Reply:
It's really useless for us. here is to define some test common methods to test on the chain.
https://github.com/paritytech/parity-bridges-common/blob/e2e283027294a856663a6f4ad5472808c5a24d49/bin/millau/runtime/Cargo.toml#L78

Reply:
Then make a clean about this crate?

Reply:
https://github.com/darwinia-network/darwinia-messages-substrate/pull/149/commits/dd487ecb39be16cd567f074e15bab55c6064a30e